### PR TITLE
Add extension buttons

### LINKED ISSUE #18: Have posibility in RM to complete set of records
Image

Roles:
- OU - operator regular user
- OAU - operator admin user
- SAU - supplier admin user 

Scenario is:
1) OU edits + save records -- he can see in list of records that this record is not complete
2) OU complete-s records -- he hits a button next to Save button called "Complete/Close record"
3) OAU can see which records are completed/not completed and there is a button "Publish" to publish all the completed records
4) SAU - opens records sent by OAU (BTW, all of the records are again not completed)
5) SAU edits + save records -- he can see in list of records that this record is not complete
6) complete-s/reject records -- he hits a button next to Save button called "Complete/Close record" or "Reject record"
3) OAU can see which records are **completed/not completed/rejected** -- i would call completed --> accepted

The idea:

Image

OU/OAU will see:

Image

SAO will see:

Image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #18 

@blcham Just putting here what was done so far if you want to deploy to at least have the UI of the buttons

### CODE REVIEW

File: js/i18n/cs.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ export default {
         'please-wait': 'Prosím, čekejte...',
         'actions': 'Akce',
         'required': 'Políčka označená * jsou povinná',
+        'reject': 'Odmítnout',
+        'accept':'Přijmout',
+        'complete': 'Dokončit',
+        'publish': 'Publikovat',
```
Comment (LaChope) [inline js/i18n/cs.js:39]: @blcham Can you verify that the Cwech is correct? :)

**Comment by LaChope** [comment]
@blcham All described scenarios in #18 are now covered

**Comment by blcham** [comment]
!!! DO NOT MERGE THIS PR !!! It has disabled tests @LaChope

**Comment by LaChope** [comment]
@blcham can be closed in favor of #50

**Comment by blcham** [comment]
Closing as this PR was replaced by cleaner https://github.com/kbss-cvut/record-manager-ui/issues/49.

**Comment by blcham** [comment]
Closing in favor of cleaner PR that does the same https://github.com/kbss-cvut/record-manager-ui/pull/53.

**Pull request closed without merging**